### DISCUSSION TITLE
Felipe Huici @ Unikraft GmbH, EMEA

### DISCUSSION THREAD

**Discussion by felipehuici** [Green-Software-Foundation/discussions#37] [Introductions]
Hi all! 👋

I'm CEO & Co-Founder at Unikraft, a startup building extremely lightweight and green virtualization solutions for the cloud based on tech called unikernels. You can think of unikernels as extremely specialized images tailored to the needs of an application. For instance, instead of deploying a Ubuntu or Docker image consisting of hundreds of MBs just to run NGINX, you can do so with Unikraft in an image of a couple of MBs only. Memory consumption is also much lower, as are boot times, storage reqs, efficiency in terms of throughput, etc. All of this leads to fewer resources needed and, of course, a smaller carbon footprint 🌿. Unikraft is an open core company, the accompanying Linux Foundation open source project is hosted at www.unikraft.org .

Given the above, I suppose my interest in the Green Software Foundation might be obvious, though since I'm completely new here I would love some pointers as to whether what we're working on sounds useful/applicable, and if so, how we could get involved.

And looking forward to connecting with everybody, my LinkedIn is: https://www.linkedin.com/in/felipe-huici

Excited to have found this community! 😀

**Comment by NAMRATA-WOKE** [comment]
@felipehuici, welcome! Since there is more activity in the other [Welcome Discussion](https://github.com/orgs/Green-Software-Foundation/discussions/2), you might want to reintroduce yourself there! 
Don't forget to hit the subscribe button to get notified of new activity!
